### DISCUSSION TITLE
Amasty Improved Navigation

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#31] [3. Integrations]
Hi There,

I am working on Magento2 since last 3-4 years. I implemented and integrated the various extensions on the many magento Websites.

Now I am working on the Magento website having the Swissup Breeze theme integrated. But As per the requirement I can not achieve to integrate the Third party Amasty extension on the website due to JS errors.

I go through the Breeze theme document as well but getting difficulties to modify it. 

Also this type of modules usually used in earlier websites. So I suggest you to integrate this Module (Amasty Shopby) into the Swissup Breeze theme. 

Hope this will help. Please let me know if you have any questions

Thanks
Rutul

**Comment by dimdimich** [comment]
Can you please open ticket at https://swissuplabs.com/contacts/ so we can discuss possible options for this integration?

**Reply by rutulritz** [reply]
I created a ticket,  but not found any ticket number at all.

**Comment by dimdimich** [comment]
Your ticket is opened as guest. You can create ticket using same email and this ticket will be linked to your account.
